apps/web/lib/daily-webhook/getCalendarEvent.ts

```diff
@@ -0,0 +1,40 @@
+import { getTranslation } from "@calcom/lib/server/i18n";
+import type { CalendarEvent } from "@calcom/types/Calendar";
+
+import type { getBookingResponse } from "./getBooking";
+
+export const getCalendarEvent = async (booking: getBookingResponse) => {
```

Comment:
This one seems like a very common requirement. So, I think it should exist as packages/lib/buildCalendarEventFromBooking or similar. So, that others start using it.

Reply:
other files like handleNewBooking need much more properties to create calendar event. so it's not really reusable

Reply:
It still could be made reusable.
1. Some other usecase might have the requirement of exact same props.
2. If handleNewBooking needs more props, it can have its own function but that function can still use this function and add more props to it.

Also, if two calendarEvents are using different props, I think we should name them differently. A calendar event should be consistent in my opinion.